Test.c

```diff
@@ -4,11 +4,139 @@
 // Ayisha Sameera,GET.
 // ------------------------------------------------------------------
 // Test.c
-// Program on branch main
+// Program on branch A5
 // ------------------------------------------------------------------------------------------------
-
+#pragma warning(disable:4996)
+#include "Header.h"
 #include <stdio.h>
+#include <stdbool.h>
+#include <malloc.h>
+#include <string.h>
+#include <stdlib.h>
+
+typedef struct {
+   int error;            // Stores the error code.
+   int num;              // Stores the input value.
+}Type;
+
+/// <summary>Windows OS specific command to clear console screen.</summary>
+static void ClearScreen () {
+#ifdef _WIN32
+   system ("cls");
+#endif
+}
+
+/// <summary>Checks if the array is in ascending order.</summary>
+bool Check (int arr[], int length) {
+   for (int i = 1; i < length; i++) if (arr[i - 1] > arr[i]) return false;
+   return true;
+}
+
+void PrintArray (int arr[], int size, int expLength) {
+   printf ("      "YELLOW"Input\t"RESET"|");
+   for (int i = 0; i < size; i++) printf ("%12d", arr[i]);
+   for (int i = expLength - size; i > 0; i--)printf ("%12s", "");
+   HeapSort (arr, size);

```

Comment:
Print Array method should only contain methods to print or display elements, sorting should be performed outside